ironfish/src/metrics/meter.ts

```diff
@@ -13,27 +13,36 @@ import { RollingAverage } from './rollingAverage'
  * completd each tick cycle and record that in various
  * rolling averages.
  *
- * @TODO: Move RollingAverages to exponentially-weighted moving average (EWMA)
  * */
 export class Meter {
   private _started = false
-  private _rate1s: RollingAverage
-  private _rate5s: RollingAverage
-  private _rate1m: RollingAverage
-  private _rate5m: RollingAverage
-  private _average: RollingAverage
+  private _rate1s: EwmaAverage
+  private _rate5s: EwmaAverage
+  private _rate1m: EwmaAverage
+  private _rate5m: EwmaAverage
+  private _average: EwmaAverage
   private _count = 0
   private _interval: SetIntervalToken | null = null
   private _intervalMs: number
   private _intervalLastMs: number | null = null
 
+  private _halfLife = 1
+
+  private readonly METER_TIME_INTERVALS_MS = {
+    rate1s: 1000,
+    rate5s: 5000,
+    rate1m: 1 * 60 * 1000,
+    rate5m: 5 * 60 * 1000,
+    average: 100,
+  }
+
   constructor() {
     this._intervalMs = 1000
-    this._rate1s = new RollingAverage(1000 / this._intervalMs)
-    this._rate5s = new RollingAverage(5000 / this._intervalMs)
-    this._rate1m = new RollingAverage((1 * 60 * 1000) / this._intervalMs)
-    this._rate5m = new RollingAverage((5 * 60 * 1000) / this._intervalMs)
-    this._average = new RollingAverage(100)
+    this._rate1s = new EwmaAverage(this._halfLife)
+    this._rate5s = new EwmaAverage(this._halfLife)
+    this._rate1m = new EwmaAverage(this._halfLife)
+    this._rate5m = new EwmaAverage(this._halfLife)
+    this._average = new EwmaAverage(this._halfLife)
```

Comment:
I'm not sure this is correct, I commented about this here. https://github.com/iron-fish/ironfish/pull/1646#discussion_r908798362